.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,29 @@
+# Upload a Python package when a release is created
+# https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows
+
+name: Publish Python 🐍 distributions 📦 to PyPI
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build-n-publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+
+      - name: Build a source tarball and a binary wheel
+        # https://pypa-build.readthedocs.io
+        run: |
+          python -m pip install build
+          python -m build --sdist --wheel
+
+      - name: Publish 📦 to PyPI
+        # https://github.com/pypa/gh-action-pypi-publish
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          password: ${{ secrets.PYPI_API_TOKEN }}
+          verbose: true
```

Comment:
will only we be able to see the output from this step?

also, whose API token is set here?

Reply:
> will only we be able to see the output from this step?

From what I checked (for [example](https://github.com/systemd/systemd/actions)) everyone can see the logs.

It [adds](https://github.com/pypa/gh-action-pypi-publish/blob/master/twine-upload.sh#L44) `--verbose` to `twine upload` and I don't think `twine` is so stupid to output the password even with `--verbose`.
But if it is, I'll simply revoke the password and remove this flag.

> whose API token is set here?

Good question. [usercont-release-bot](https://pypi.org/user/usercont-release-bot) and I've been thinking about whether to use my account or create a completely new "packit-bot" account.
Also, I know I should comment it somewhere, but not sure where, I wish there was some possibility to add a comment to [secrets](https://github.com/packit/ogr/settings/secrets/actions).

Reply:
Added the comment. Can someone approve?